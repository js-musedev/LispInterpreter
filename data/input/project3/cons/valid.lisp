(CONS (PLUS 2 3) (CONS 8 NIL))
(CONS 2 (CONS 3 (CONS 4 (CONS 5 NIL))))
(CONS 1 (CONS 3 NIL))
(ATOM (CDR (CONS 34 (CONS 54 NIL))))
(CONS (PLUS 1 2) (CONS (PLUS 3 3) NIL))
(CONS (CAR (CONS 7 (CONS 8 NIL))) (CDR (CONS 6 NIL)))
