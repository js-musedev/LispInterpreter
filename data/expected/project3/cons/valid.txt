(5 8)
(2 3 4 5)
(1 3)
F
(3 6)
(7)